lib/services/plugins-service.ts

```diff
@@ -90,16 +95,16 @@ export class PluginsService implements IPluginsService {
 				}
 			}
 
-			var parts = pluginName.split("@");
-			pluginName = parts[0];
-			var version = parts[1] || "";
-
-			if(this.getPluginByName(pluginName).type === pluginsDataLib.PluginType.MarketplacePlugin && !version) {
+			var pluginToAdd = this.getPluginByName(pluginName);
+			if(pluginToAdd.type === pluginsDataLib.PluginType.MarketplacePlugin) {
 				var versions = this.getPluginVersions(pluginName);
-				version = this.promptForVersion(pluginName, versions).wait();
+				if(version && !_.any(versions, v => v.value === version)) {
+					this.$errors.fail("Invalid version %s. The valid versions are: %s", version, versions.map(v => v.value).join(", "));
```

Comment:
failWithoutHelp maybe? (not sure, maybe it's good to have the help here, so the user will understand from where to see available versions)